Makefile

```diff
@@ -74,10 +76,18 @@ endif
 	@printf "The engine has been cleaned.\n"
 
 version: variables
+ifneq ($(OVERWRITE_VERSION),"True")
+	@printf "Setting mod version to $(MOD_VERSION)\n"
+else 	
 	@awk '{sub("Version:.*$$","Version: $(VERSION)"); print $0}' $(MANIFEST_PATH) > $(MANIFEST_PATH).tmp && \
 	awk '{sub("/[^/]*: User$$", "/$(VERSION): User"); print $0}' $(MANIFEST_PATH).tmp > $(MANIFEST_PATH) && \
 	rm $(MANIFEST_PATH).tmp
-	@printf "Version changed to $(VERSION).\n"
+ifeq ("$(MOD_VERSION)", "$(VERSION)")
```

Comment:
$(MOD_VERSION) gets overridden and modversion and tag are equal after the first buildpackage.sh has run.

Reply:
Can you note this in a comment?

Reply:
Done.